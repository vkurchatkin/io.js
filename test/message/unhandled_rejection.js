var common = require('../common');

Promise.reject(new Error('error'));
