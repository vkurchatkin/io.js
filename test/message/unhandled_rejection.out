node.js:*
          throw value;
                ^
Error: error
    at Object.<anonymous> (*test*message*unhandled_rejection.js:*)
    at Module._compile (module.js:*)
    at Object.Module._extensions..js (module.js:*)
    at Module.load (module.js:*)
    at Function.Module._load (module.js:*)
    at Function.Module.runMain (module.js:*)
    at startup (node.js:*)
    at node.js:*
